packages/main/src/ColorPalette.ts

```diff
@@ -357,6 +370,11 @@ class ColorPalette extends UI5Element {
 	_onColorContainerKeyDown(e: KeyboardEvent) {
 		const target = e.target as ColorPaletteItem;
 		const lastElementInNavigation = this.colorPaletteNavigationElements[this.colorPaletteNavigationElements.length - 1];
+
+		if ((isUp(e) || isDown(e)) && this._currentlySelected !== undefined && this.colorPaletteNavigationElements.includes(this._currentlySelected)) {
+			this._currentlySelected = undefined;
```

Comment:
set to null, as it is runtime. Try using null when you set something to e falsy value, as null is an assigned value meaning nothing and undefined means a variable has been declared but not defined yet.